BusyList/TaskItem.cs

```diff
@@ -8,21 +8,23 @@ public TaskItem(int id, string description, PriorityEnum priority = PriorityEnum
         {
             Id = id;
             Description = description;
-            Status = status;
+            TaskStatus = status;
+            Priority = priority;
         }
         public int Id { get; }
 
         public string Description { get; set; }
-
-        public TaskStatus Status { get; set; }
+        public TaskStatus TaskStatus { get; set; }
+        public PriorityEnum Priority { get; set; }
```

Comment:
Naming Priority would be better.
Chris suggested this on EasyList: https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md